packages/KMenu/KMenuItem.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div
+    :data-testid="item ? `${item.title.replace(' ', '-')}-menu-item` : 'menu-item'"
+    :test-mode="testMode"
+    :class="[isOpen && expandable ? 'title-dark' : '', {'expand-item' : expandable}]"
+    class="k-menu-item">
+    <div
+      class="menu-title"
+      @click="isOpen = !isOpen">
+      <span
+        :class="isOpen && expandable ? 'title-dark' : ''"
+        class="span-menu-title">
+        <slot name="itemTitle">
+          {{ item ? item.title : '' }}
+        </slot>
+      </span>
+      <span
+        class="span-icon-container">
+        <KIcon
+          v-if="expandable"
+          :icon="isOpen ? 'chevronUp' : 'chevronDown'"
+          color="var(--grey-400)"
+          size="16"
+        />
+      </span>
+    </div>
+    <div
+      :class="isOpen ? 'd-flex' : 'd-none'"
+      class="menu-content">
+      <slot name="itemBody">
+        <div
+          v-if="(type === 'string' || 'divider') && expandable">
+          {{ item ? item.description : '' }}
+        </div>
+        <div
+          v-else-if="(type === 'number' || 'divider') && expandable">
+          {{ item ? item.description : '' }}
+        </div>
+      </slot>
+    </div>
+    <slot
+      v-if="type === 'divider'">
+      <KMenuDivider />
+    </slot>
+  </div>
+</template>
+
+<script>
+import KEmptyState from '@kongponents/kemptystate/KEmptyState.vue'
+import KMenuDivider from './KMenuDivider.vue'
+import KIcon from '@kongponents/kicon/KIcon.vue'
+
+export default {
+  name: 'KMenuItem',
+  components: { KMenuDivider, KIcon, KEmptyState },
+  props: {
+    item: {
+      type: Object,
+      default: null
+    },
+    expandable: {
+      type: Boolean,
+      default: false
+    },
+    type: {
+      type: String,
+      default: 'string',
+      validator: (value) => {
+        return [
+          'string',
+          'number',
+          'divider'
+        ].indexOf(value) !== -1
+      }
+    },
+    /**
+     * Test mode - for testing only, strips out generated ids
+     */
+    testMode: {
+      type: Boolean,
+      default: false
+    }
+  },
+
+  data: () => {
+    return {
+      isOpen: false
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.k-menu-item {
+  list-style: none;
+  margin: 0;
+  white-space: nowrap;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 24px;
+  color: var(--grey-500);
+  position: relative;
+  padding-left: 24px;
+  padding-top: 20px;
+  &:not(:last-child):after {
+    content: " ";
+    position: absolute;
+    left: 56%;
+    transform: translateX(-57%);
+    bottom: 0;
+    width: 88%;
+    height: 1px;
+    background-color: var(--grey-300);
+  }
+}
+
+.k-menu ul:not(:last-child):after {
+    content: " ";
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏indentation

Reply:
Fixed!